src/validations/css/stylelint.js

```diff
@@ -0,0 +1,65 @@
+import Validator from '../Validator';
+
+const errorMap = {
+  // CSS Syntax Errors
+  'Unclosed block': (error) => {
+    const token = error.source;
+
+    return {
+      reason: 'missing-closing-curly',
+      payload: {error: token},
+    };
+  },
+
+  // Custom Rules
+  'declaration-block-trailing-semicolon': (error) => {
+    const token = error.css;
+
+    return {
+      reason: 'missing-semicolon',
+      payload: {error: token},
+    };
+  },
+};
+
+class StyleLintValidator extends Validator {
+  constructor(source) {
+    super(source, 'css', errorMap);
+  }
+
+  _getRawErrors() {
+    return System.import('../linters').then(
+      ({stylelint}) => stylelint(
+        this._source
+      ).then(
+        (result) => ({result})
+      ).catch(
+        (syntaxError) => ({syntaxError})
+      )
```

Comment:
Also, I think this may work by something of a coincidence—`_getRawErrors()` is expected to return a promise that resolves to an array, not to an object.